task3-4/task3-4/RoundedProgressBar/RoundedProgressBar.swift

```diff
@@ -0,0 +1,146 @@
+//
+//  File.swift
+//  task3-4
+//
+//  Created by Григоренко Александр Игоревич on 06.10.2022.
+//
+
+import UIKit
+
+/// Круглый  прогресс бар заполняющийся с течением времени.
+class CircularProgressView: UIView {
+
+    private var progressLayer = CAShapeLayer()
+    private var trackLayer = CAShapeLayer()
+    private var didConfigureLabel = false
+    private var rounded: Bool
+    private var filled: Bool
```

Comment:
```suggestion
    private var isFilled: Bool
```
Видишь бул ставь is приставкой